openfga/v1/errors_ignore.proto

```diff
@@ -71,6 +71,14 @@ enum ErrorCode {
   unsupported_schema_version = 2057;
 }
 
+enum ResourceExhaustedErrorCode {
```

Comment:
Need to change this to UnprocessableContentErrorCode to avoid conflict if other users of OpenFGA that uses OpenFGA as library also has ResourceExhaustedErrorCode